### ISSUE TITLE
Are there plans on updating the material design to Google's newer concept?

### ISSUE THREAD

**Issue by ChildishGiant** [Dogfalo/materialize#6004]
So google seem to be adopting a newer style of material design (Known as the "Material Design Refresh") and we can see it coming up in apps like [Google News](https://news.google.com), [Google Podcasts](https://play.google.com/store/apps/details?id=com.google.android.apps.podcasts&hl=en_US), android messages on both [android](https://play.google.com/store/apps/details?id=com.google.android.apps.messaging) *and* [web](https://www.theverge.com/2018/6/19/17479554/android-messages-how-to-text-from-web-feature)
, the [gmail redesign](https://www.theverge.com/2018/4/12/17227974/google-gmail-design-features-update-2018-redesign) and even [chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=822069#c12). I could go on for hours. 
![image](https://user-images.githubusercontent.com/13716824/41884438-6af70cf2-78eb-11e8-81f9-6cf80bc7d175.png)

So my question is whether or not there are plans to integrate this *refresh* into materialize,  potentially as an alternate stylesheet? There currently aren't any firm specs so I know that it's not possible to implement currently but I'm wondering if it's on the roadmap 😄

**Comment by ChildishGiant** [comment]
Yea thanks :) Would the current style only be available in past versions then? I imagine if people want it, the community could upkeep a separate branch if this replaces the current style.

**Comment by Dogfalo** [comment]
I don't think that the updated styles will create that much divide in the community, but we will see.
